### ISSUE TITLE
[ISO] Wrong ISO link for CLI guide on setup:static-content:deploy

### ISSUE THREAD

**Issue by kevingimbel** [magento/devdocs#1162]
Hey,

in the guide for static content deployment (http://devdocs.magento.com/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html) you link to the external www.loc.gov site for ISO Codes. It states that the `-language|-l` CLI flag uses the ISO-636 codes but the link goes to ISO-639 (https://www.loc.gov/standards/iso639-2/php/code_list.php). ISO-639 does not seem to be the correct language code.

When I search for ISO-636 I don't really find anything usefull beside a issue thread on GitHub, https://github.com/magento/magento2/issues/8398#issuecomment-277088868, where @hostep says

> And you are definitely correct, that ISO-636 is very incorrect.

So which language code is it after all? @hostep says in the comment its kind of a mix between ISO-639-1 and ISO-3166-2? If it's not a standard but in fact a _mix of standards_ you may want to maintain your own list in the docs instead of linking to the loc.gov site - or just say "run `info:language:list `". When I follow the link and copy the code, e.g. for German, I only get an error and the note to look at `info:language:list`.

```sh
$ bin/magento setup:static-content:deploy de
                                              
  [InvalidArgumentException]
  de argument has invalid value, please run info:language:list for list of available locales  
```
`bin/magento info:language:list` does return a (correct) list of language codes in the format of `en_US`, `ar_KW`, `de_DE`, etc. These codes look more like a variant of [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag) (`-` replaced by `_`). 

So, tl;dr, what kind of language code standard is Magento using? IMO there's no point in linking to a standard when the standard isn't even used. The link should be removed.

**Comment by jeff-matthews** [comment]
@jcalcaben, please remove references to ISO standards.
